test/end-to-end/offer.js

```diff
@@ -2883,6 +2883,15 @@ describe("Offer endpoint tests", () => {
                         .expect(HTTPStatus.OK);
                     expect(res.body).toHaveProperty("applyURL", applyURL);
                 });
+
+                test("Should edit if applyURL is specified as null", async () => {
+                    const applyURL = null;
+                    const res = await test_agent
+                        .post(`/offers/edit/${future_test_offer._id.toString()}`)
+                        .send(withGodToken({ applyURL }))
+                        .expect(HTTPStatus.OK);
+                    expect(res.body).toHaveProperty("applyURL", applyURL);
```

Comment:
Just out of curiosity: why is this test different from the others?

Reply:
The `applyURL` tests had that structure so I just followed it :shrug: Nothing that really matters